easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'io_lib'
+version = '1.14.8'
+
+homepage = 'http://sourceforge.net/projects/staden/files/io_lib/'
+description = """Io_lib is a library of file reading and writing code to provide a general purpose trace file (and Experiment
+ File) reading interface. The programmer simply calls the (eg) read_reading to create a "Read" C structure with the data loaded
+ into memory. It has been compiled and tested on a variety of unix systems, MacOS X and MS Windows. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [('http://sourceforge.net/projects/staden/files/%(namelower)s/%(version)s', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
+sanity_check_paths = {
+    'files': ["bin/scramble", "bin/append_sff", "bin/ztr_dump"],
+    'dirs': [],
+}
+
+moduleclass = 'bio'
```

Comment:
eh, is everything 'bio' nowadays?! :)
although this is really a fork of code out of `staden` bioinfo package, I'd suggest to reconsider the category...